### ISSUE TITLE
Cannot build non-release images

### ISSUE THREAD

**Issue by mmilata** [openshift/origin#12581]
##### Version
git master / 1ccbb6cdcb6bbc8539f3bd2f43970846db90be6f

##### Steps To Reproduce
```
$ git checkout master
$ git reset --hard origin/master
$ make clean
$ make
++ Building go targets for linux/amd64: cmd/openshift cmd/oc pkg/sdn/plugin/sdn-cni-plugin vendor/github.com/containernetworking/cni/plugins/ipam/host-local vendor/github.com/containernetworking/cni/plugins/main/loopback
++ Placing binaries
hack/build-go.sh took 688 seconds
$ OS_RELEASE=n hack/build-images.sh
Building images from source v1.5.0-alpha.2-1ccbb6c-45:

hack/build-images.sh: line 23: OS_IMAGE_COMPILE_GOFLAGS: unbound variable
```
I tried to workaround the unbound variables but there are further errors like the pod binary not being built. The same steps worked couple of days ago, though.

**Comment by stevekuznetsov** [comment]
Fixed in #12548

**Comment by mmilata** [comment]
@stevekuznetsov tested the patch, now I'm getting:
```
[root@localhost origin]# OS_RELEASE=n hack/build-images.sh 
Building images from source v1.5.0-alpha.2-a63a831-47-dirty:

/data/src/github.com/openshift/origin/hack/common.sh: line 295: platforms[@]: unbound variable
[ERROR] PID 5293: hack/common.sh:238: `( os::build::internal::build_binaries "${binaries[@]+"${binaries[@]}"}" )` exited with status 1.
[INFO]          Stack Trace: 
[INFO]            1: hack/common.sh:238: `( os::build::internal::build_binaries "${binaries[@]+"${binaries[@]}"}" )`
[INFO]            2: hack/common.sh:227: os::build::build_binaries
[INFO]            3: hack/build-images.sh:23: os::build::build_static_binaries
[INFO]   Exiting with code 1.
```

**Comment by stevekuznetsov** [comment]
@mmilata that's fixed in #12561

**Comment by stevekuznetsov** [comment]
Fixes merged.
